src/functions-and-arrays.js

```diff
@@ -1,41 +1,76 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
-
+function maxOfTwoNumbers(a, b) {
+  if (a > b) {
+    return a;
+  } else if (b > a) {
+    return b;
+  } else {
+    return a || b;
```

Comment:
This will not behave as you may expect. It will return `a` all the time unless` a = 0` because 0 evaluates to a falsy value and any other value evaluates to true. In this case, it will return `b`. It will still pass the test case because we will not reach this line of code unless a and b are equal, so it won't matter.

If you want to handle the equality case, could change line 3 to be `if(a >= b)` and then it returns `a` if `a` and `b` are equal.

You could do this in a shorter version. Check the lab solution posted by Jan.